scripts/executors/src/deprecate-react-components-preview-packages.spec.ts

```diff
@@ -0,0 +1,152 @@
+import childProcess from 'node:child_process';
+import fs from 'node:fs';
+import path from 'node:path';
+
+import monorepoScripts from '@fluentui/scripts-monorepo';
+import * as devkit from '@nx/devkit';
+import type { ChangeType } from 'beachball';
+
+import { deprecatePackage, getPackagesToDeprecate } from './deprecate-react-components-preview-packages';
+
+const mockExecSync = jest.spyOn(childProcess, 'execSync');
+const mockReaddirSync = jest.spyOn(fs, 'readdirSync');
+const mockReadJsonFile = jest.spyOn(devkit, 'readJsonFile');
+const mockJoin = jest.spyOn(path, 'join');
+const mockGetAllPackageInfo = jest.spyOn(monorepoScripts, 'getAllPackageInfo');
+
+describe('deprecatePackage', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should execute npm deprecate command', () => {
+    mockExecSync.mockImplementation(() => '');
+
+    deprecatePackage('test-package', 'npm-token');
+
+    expect(mockExecSync).toHaveBeenCalledWith(
+      'npm deprecate test-package "Deprecated in favor of stable release" --registry https://registry.npmjs.org/ --//registry.npmjs.org/:_authToken=npm-token',
+      { stdio: 'inherit' },
+    );
+  });
+
+  it('should log error when deprecation fails', () => {
+    mockExecSync.mockImplementationOnce(() => {
+      throw new Error('Deprecation failed');
+    });
+
+    expect(() => deprecatePackage('test-package', 'npm-token')).toThrowError(
```

Comment:
this would fail master on `lint` - please update your PR branch

`toThrowError` is deprecated

```suggestion
    expect(() => deprecatePackage('test-package', 'npm-token')).toThrow(
```